Makefile

```diff
@@ -24,10 +24,16 @@ PINNED_DEPENDENCIES := \
 
 ##### Build #####
 
-build:
-	@printf $(COLOR) "Build Temporal CLI with OS: $(GOOS), ARCH: $(GOARCH)..."
+build: build-cli build-doc-gen
+
+build-cli:
```

Comment:
Why do we even need a makefile in this repo? Doesn't seem like there's that much value.

Reply:
done

Reply:
sure removed. I guess mostly just to have makefile consistent with the tools in the repo